tests/PHPStan/Command/CommandHelperTest.php

```diff
@@ -90,6 +90,32 @@ public function dataBegin(): array
 				],
 				false,
 			],
+			[
+				'',
+				'',
+				__DIR__ . '/data/config.php',
+				null,
+				[
+					'level' => '3',
+				],
+				false,
+			],
+			[
+				'',
+				'Invalid .php project config given: Nette\DI\Config\Adapters\PhpAdapter::load(): Return value must be of type array, int returned',
```

Comment:
explicit new error, when .php files used as config do not provide a proper format